slack_approval/functions/requirements-provision.txt

```diff
@@ -1,3 +1,3 @@
 goblet-gcp==0.9.2
 slack_sdk==3.18.1
-git+https://github.com/premisedata/slack-approval.git
\ No newline at end of file
+slack-approval @ git+https://github.com/premisedata/slack-approval.git@INFRA-1497-Slack-approvals-ability-to-modify-request-in-slack
```

Comment:
dont change this. we should change to the latest release actually.

Reply:
changed